packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -1028,6 +1028,30 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
     return tableWidth;
   }
 
+  double _pinnedDataHeight(BoxConstraints tableConstraints) => min(
+        widget.rowItemExtent! * pinnedData.length,
+        tableConstraints.maxHeight / 2,
+      );
+
+  int _dataRowCount(
+    BoxConstraints tableConstraints,
+    bool showColumnGroupHeader,
+  ) {
+    if (!widget.fillWithEmptyRows) {
+      return _data.length;
+    }
+
+    var maxHeight = tableConstraints.maxHeight;
+    final columnHeadersCount = showColumnGroupHeader ? 2 : 1;
+    maxHeight -= columnHeadersCount * areaPaneHeaderHeight;
+
+    if (pinnedData.isNotEmpty) {
+      maxHeight -= _pinnedDataHeight(tableConstraints);
```

Comment:
The height of the divider between the pinned data and the rest of the data that needs to be accounted for. Looks like we are using `const ThickDivider(),` below

Reply:
updated